src/Logging/Logger.php

```diff
@@ -112,6 +131,27 @@ public static function emergency(string $sMessage, LogContext $logContext, array
 
     private static function logContext(string $sLevel, string $sMessage, LogContext $logContext, array $aMetadata = []): void
     {
-        self::log($sLevel, $sMessage, \array_merge([self::CONTEXT => (string)$logContext], $aMetadata));
+        try {
+            $aDebugBacktrace = \debug_backtrace();
+            list($aTraceOfLogCall, $aTraceBeforeLogCall) = [...\array_filter(
```

Comment:
IMHO `array_shift($aDebugBacktrace);` is more readable.

Reply:
why not